File: decompiled/sources/com/streamax/config/db/DatebaseConfig.java
package com.streamax.config.db;

public interface DatebaseConfig {
    public static final String DatabaseName = "DeviceInfo.db";
    public static final String TableName = "DeviceInfo";

    public interface DeviceInfoTable {
        public static final String dbCurrentChannel = "dbCurrentChannel";
        public static final String dbCurrentDay = "dbCurrentDay";
        public static final String dbCurrentPlan = "dbCurrentPlan";
        public static final String dbCurrentTotalChannel = "dbCurrentTotalChannel";
        public static final String dbCurrentTotalPlan = "dbCurrentTotalPlan";
        public static final String dbDeviceIp = "dbDeviceIp";
    }
}

File: decompiled/sources/com/streamax/config/db/DatebaseConfig.kt
package com.streamax.config.db

interface DatebaseConfig {
    companion object {
        const val DatabaseName: String = "DeviceInfo.db"
        const val TableName: String = "DeviceInfo"
    }

    interface DeviceInfoTable {
        companion object {
            const val dbCurrentChannel: String = "dbCurrentChannel"
            const val dbCurrentDay: String = "dbCurrentDay"
            const val dbCurrentPlan: String = "dbCurrentPlan"
            const val dbCurrentTotalChannel: String = "dbCurrentTotalChannel"
            const val dbCurrentTotalPlan: String = "dbCurrentTotalPlan"
            const val dbDeviceIp: String = "dbDeviceIp"
        }
    }
}
